app/models/profile.rb

```diff
@@ -34,7 +21,23 @@ def activate
     end
   end
 
-  def verified?
-    verified_at.present?
+  def plain_pii
+    @_plain_pii ||= Pii::Attributes.new
```

Comment:
Let's address this later, but wanted to comment before I forget:

I'm not sure we should memoize this method in this way -- I think that this would allow pretending to read PII attributes without actually decrypting them. IMO if a profile has PII but it has not been decrypted, this should be nil or something so we get errors when try to access fields.